wpilibc/src/main/native/include/Buttons/POVButton.h

```diff
@@ -0,0 +1,20 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
This shim is unnecessary because the POVButton class didn't exist in 2018; there's nothing to move.